docs/ruleset_functions/get_rmr_scc_skylight_roof_ratios.md

```diff
@@ -1,15 +1,15 @@
 
-## get_building_scc_skylight_roof_ratios
+## get_rmr_scc_skylight_roof_ratios
 
-Description: This function would determine skylight and envelope roof ratios for a building for residential, non-residential, mixed and semi-heated surface conditioning categories.  
+Description: This function would determine skylight and envelope roof ratios for a RMR for residential, non-residential, mixed and semi-heated surface conditioning categories.  
 
 Inputs:
 
   - **RMR**: The RMR that needs to determine skylight and envelope roof ratios.  
 
 Returns:
 
-- **building_scc_skylight_roof_ratios_dictionary**: A dictionary that saves each surface conditioning category (residential, non-residential, mixed and semi-heated) with its skylight-roof-ratios for each building in RMR.
+- **building_scc_skylight_roof_ratios_dictionary**: A dictionary that saves each surface conditioning category (residential, non-residential, mixed and semi-heated) with its skylight-roof-ratios for the input RMR, e.g. {"EXTERIOR RESIDENTIAL": 0.02, "EXTERIOR NON-RESIDENTIAL": 0.05, "EXTERIOR MIXED": 0.00, "SEMI-EXTERIOR": 0.00}.
```

Comment:
I would refer to it as RulesetModelInstance, as the RMR may have more than one ASHRAE229.ruleset_model_instances